dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -514,23 +518,27 @@ internal class RumViewManagerScopeTest {
     // region AppLaunch View
 
     @Test
-    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, event is relevant }`(
+    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, any event }`(
         forge: Forge
     ) {
         // Given
         CoreFeature.processImportance = RunningAppProcessInfo.IMPORTANCE_FOREGROUND
         testedScope.applicationDisplayed = false
-        val fakeEvent = forge.validAppLaunchEvent()
+        val fakeEvent = forge.anyRumEvent()
+        val appStartTimeNs = fakeEvent.eventTime.nanoTime - TimeUnit.MILLISECONDS.toNanos(500)
```

Comment:
or we can a bit more randomness and simply do `val appStartTimeNs = forge.aLong(min = 0L, max = fakeEvent.eventTime.nanoTime)`

Reply:
👍 Though I think we probably want to go min, say fakeTime - 1000ms second to fakeTime - 100ms? What do you think?

Reply:
It is ok for me. We don't have thresholds to filter out too big times, so we can accept any positive value as min (we just try to avoid hardcoded values in our tests).